### ISSUE TITLE
Statistics section in add-on details page has layout issues for German smaller screen sizes

### ISSUE THREAD

**Issue by AlexandraMoga** [mozilla/addons-frontend#4089]
Prerequisites: Locale set to Deutsch 

STR:

1. Go to AMO homepage in responsive design mode (~750 - 1000 width) 
2. Select an Add-on to view its listing page (i.e. https://addons-dev.allizom.org/de/firefox/addon/awesome-screenshot-plus-/)
3. Observe the statistics section in the right corner

Expected result:
There are no layout issues

 Actual result:
The overall rating text falls outside of the allocated area

Notes:
- reproduces on all AMO servers with FF57, Win10 and Android 7.0

 
![image](https://user-images.githubusercontent.com/31961530/34112243-a6c72440-e414-11e7-89a9-894f236e80bf.png)

**Comment by muffinresearch** [comment]
Looks like this should change to a different breakpoint sooner.

**Comment by imrjk07** [comment]
I found what to fix. But i need some assistance in making the change.

**Comment by caitmuenster** [comment]
Hi @imrjk07! Thanks for your interest. This one might be a little bit complicated if this is your first contribution. Would you be interested in getting started with #[7377](https://github.com/mozilla/addons-server/issues/7377) instead?

**Comment by imrjk07** [comment]
Hello @caitmuenster , Thanks for replying. I tried to recreate the issue #7377. But i'm not sure the link in that issue is taking me somewhere and not seeing the issue page.

**Comment by AlexandraMoga** [comment]
This might have solved the original issue but now there is a word wrapping problem. Since we are still having a layout issue, I'm reopening this case.

![image](https://user-images.githubusercontent.com/31961530/35503636-60cf9954-04e9-11e8-85ab-090cd0bd7e07.png)

**Comment by tofumatt** [comment]
Those words are too large for the space, so they’re wrapping. I think that’s intended. I suppose a new issue to add a hyphen might be good, but I think this works as intended.

- Matt (Sent from mobile)

**Comment by AlexandraMoga** [comment]
As advised, I opened a new issue because there might be a way to avoid displaying words like this in statistics section
